### ISSUE TITLE
Reproduce issue

### ISSUE THREAD

**Issue by M3Dade** [si0wang/ThinkLite-VL#9]
Thanks for your pretty work.

I am very interested in reproducing the training results presented in your paper. We have used the russwang/ThinkLite-VL-hard-11k dataset to conduct GRPO training on Qwen2.5VL using ms-swift, with 32 completions and a learning rate of 1e-6. However, we have noticed a significant gap between our results and those reported in your publication.

If possible, could you kindly share more details regarding your training setup? For example, information about hyperparameters, batch size, number of training steps, and any preprocessing or post-processing steps would be extremely helpful for us to better understand and reproduce your results.

Thank you very much for your time and assistance!

Here is my result and code:

![Image](https://github.com/user-attachments/assets/d67493e4-556e-44eb-957e-b2da0d0b3baa)

```
CFG=qwen2.5vl-fulltune-ThinkLite-VL-hard-11k-32generations-2epoch-boxed
MAX_PIXELS=1003520 \
swift rlhf \
    --rlhf_type grpo \
    --model Qwen2.5-VL-7B-Instruct \
    --external_plugins examples/train/grpo/plugin/plugin.py \
    --train_type full \
    --dataset ThinkLite-VL-hard-11k/ThinkLite-VL-Hard-11k.jsonl \
    --torch_dtype bfloat16 \
    --attn_impl flash_attn \
    --system examples/train/grpo/thinklite_prompt_box.txt \
    --num_train_epochs 2 \
    --max_length 2048 \
    --per_device_train_batch_size 8 \
    --per_device_eval_batch_size 8 \
    --learning_rate 1e-6 \
    --save_total_limit 2 \
    --logging_steps 1 \
    --output_dir output/${CFG} \
    --gradient_accumulation_steps 1 \
    --warmup_ratio 0.05 \
    --dataloader_num_workers 4 \
    --max_completion_length 1024 \
    --reward_funcs external_thinklite_acc external_thinklite_format \
    --num_generations 32 \
    --use_vllm true \
    --vllm_gpu_memory_utilization 0.6 \
    --sleep_level 1 \
    --deepspeed zero3 \
    --log_completions true \
    --num_infer_workers 8 \
    --tensor_parallel_size 4 \
    --temperature 1.0 \
    --top_p 0.85 \
    --top_k 50 
```

![Image](https://github.com/user-attachments/assets/65f7b4c6-27d2-41bc-92e9-ce65f9479bf2)

![Image](https://github.com/user-attachments/assets/b9e926f0-56dd-43e6-b5c1-d1db9bf62070)

![Image](https://github.com/user-attachments/assets/e3e71904-ac40-43ec-ab57-2aa9e7e81661)

![Image](https://github.com/user-attachments/assets/3cc3eb06-3abc-4ec4-97e1-2aaa53ffa060)

![Image](https://github.com/user-attachments/assets/c532d516-ac60-4d85-81f0-1b932af3f400)

kl loss 有非常剧烈的抖动，请问这种正常吗
accuracy std 出现上升而不是下降，请问这种正常吗

**Comment by si0wang** [comment]
Hi, sorry for the late reply, I was quite busy last month. We found there is a format issue in our released dataset, if you use it directly for grpo training with EasyR1, the codeebase will not load the image and only do text-only training without showing any error. So one way to fix this is saving the images to your local machine and loading images from local during training. I hope this can solve your problem!

**Comment by ddydyd32** [comment]
Hello, could you share a bit on what reward functions you were using, e.g., the external_thinklite_acc external_thinklite_format, and examples/train/grpo/thinklite_prompt_box.txt? Thank you!
